src/Sharpliner.Model/Job.cs

```diff
@@ -3,20 +3,40 @@
 
 namespace Sharpliner.Model
 {
-    public record Job(
-        string Name,
-        string DisplayName,
-        IEnumerable<string> DependsOn,
-        string Condition,
-        JobStrategy Strategy,
-        JobPool Pool,
-        ContainerReference Container,
-        TimeSpan Timeout,
-        TimeSpan CancelTimeout,
-        IEnumerable<Variable> Variables,
-        IEnumerable<Step> Steps,
-        JobWorkspace Workspace = JobWorkspace.Outputs,
-        bool ContinueOnError = false);
 
     // TODO: missing properties Uses, Services
+    public record Job
+    {
+        public string Name { get; }
+
+        public string DisplayName { get; }
```

Comment:
Nullable? That will help us to validate.

```suggestion
        public string? DisplayName { get; }
```

Reply:
I don't understand?